examples/gcp_cloud_run/main.tf

```diff
@@ -14,7 +14,7 @@ terraform {
 locals {
   envs = var.environment_variables != null ? var.environment_variables : [
     {
-      name = upper("GCP_PROJECT"),
+      name  = upper("GCP_PROJECT"),
```

Comment:
```suggestion
      name = upper("GCP_PROJECT"),
```

Reply:
The default formatter changed this, and it's changing the suggestion back to what's commited